frontend/src/app/main/ui/workspace/sidebar/options/rows/color_row.scss

```diff
@@ -4,15 +4,20 @@
 //
 // Copyright (c) KALEIDOS INC
 
-@use "refactor/common-refactor.scss" as deprecated;
+@use "ds/typography.scss" as t;
+@use "ds/_sizes.scss" as *;
+@use "ds/_borders.scss" as *;
+@use "ds/mixins.scss" as *;
+@use "ds/_utils.scss" as *;
 
 .color-data {
-  @include deprecated.flexRow;
+  --reorder-left-position: calc(-1 * var(--sp-m) - var(--sp-xxs));
```

Comment:
This is a rare calculation, css variables are not supposed to be combined unless its final value is dependant on any of these variables.
Ensure that even if the variable values change, the result is still valid, otherwise this is a flaky use of variables and wold be better to use a static property .